modules/ROOT/pages/manage-backup-restore.adoc

```diff
@@ -105,13 +122,51 @@ To restore a cluster on Runtime Fabric, create a cluster *without* running the `
 ----
 kubectl scale --replicas=0 -n rtf deployment.apps/agent
 ----
-
++
+[start =5]
++
 . Restore the cluster from the backup: 
 [source,copy]
 ----
 ./rtfctl restore <path_to_backup_file>
 ----
-+
 This process may require several minutes to complete.
 
-For Runtime Fabric, confirm the `rtfctl` binary is present in the current directory and the Kubernetes (kubectl) context is set to the cluster you are restoring to.
\ No newline at end of file
+For Runtime Fabric, confirm the `rtfctl` binary is present in the current directory and the Kubernetes (kubectl) context is set to the cluster you are restoring to.
+
+== Create a Backup and Restore for OpenShift Clusters
+
+In the following example, you have a Runtime Fabric with a set of Mule applications and configurations deployed on cluster (named `cluster-1`) inside an OpenShift cluster. To create a backup and perform a restore, follow these steps:
+
+. Run the backup command: 
++
+[source,copy]
+----
+./rtfctl backup <path_to_backup_file>
+----
++
+[start=2]
+. Shutdown `cluster-1`.
+. Remove and shutdown all Mule application deployments.
```

Comment:
Run kubectl command to delete Mule applications.  Don't remove applications from Runtime Manager in UI.
ie.  kubectl delete ns <app_namespace>